etc/scripts/test-integ-pgsql.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -ex
+#
+# Copyright (c) 2018, 2020 Oracle and/or its affiliates.
```

Comment:
Copyright is wrong (as with other new files in this PR) - the year should be just `2020`

Reply:
Yes, it's copy-paste. I'll fix it in next PR.